packages/ui/sass/components/chart.scss

```diff
@@ -0,0 +1,53 @@
+%counter-shared {
+  $grey-tint: rgba(0, 0, 0, 0.25);
+  border-radius: 50%;
+  box-sizing: border-box;
+  box-shadow: 0px 1px 3px $grey-tint;
+  text-align: center;
+  font-family: $gotham;
+  letter-spacing: 2px;
+  color: #ffffff;
+  font-weight: bold;
+}
+
+.counter-default {
+  @extend %counter-shared;
+  border: 10px solid #ffffff;
+
+  &--large {
+    font-size: 54px;
+    line-height: 130px;
+    background: $gladiator;
+    width: 150px;
+    height: 150px;
+  }
+
+  &--normal {
+    font-size: 32px;
+    line-height: 85px;
+    background: $navy;
+    width: 100px;
+    height: 100px;
+  }
+
+  &--small {
+    font-size: 22px;
+    line-height: 30px;
+    background: $medium-gray;
+    width: 50px;
+    height: 50px;
+  }
```

Comment:
color and size options should be independent of each other. We should be able to pick any combination of size and color. So we will need override classes for each.